vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -102,17 +105,25 @@ public String toString() {
     public Date modifiedTime;
     public ArrayList<String> contentRating;
     public String license;
-    public com.vimeo.networking.model.Privacy privacy;
+    public Privacy privacy;
     public PictureCollection pictures;
     public ArrayList<Tag> tags;
     public StatsCollection stats;
     public Metadata metadata;
-    public com.vimeo.networking.model.User user;
+    public User user;
     private Status status;
     public VideoLog log;
     public List<Category> categories;
 
+    /**
+     * This getter is always guaranteed to return a {@link Status}, {@link Status#NONE if null}. If the Status
+     * is equal to {@link Status#TRANSCODE_STARTING} then we'll just return the Status {@link Status#TRANSCODING}
+     * since they're functionally equivalent from a client-side perspective.
+     */
     public Status getStatus() {
+        if (status == Status.TRANSCODE_STARTING) {
+            return Status.TRANSCODING;
+        }
```

Comment:
I like the addition getter methods, and I think that would solve for this nicely, since status is private.
Side note: we need to cleanup all our fields soon so that they are private with getters. setters shouldn't be needed because these should always be set just from the API.